sklearn/cross_decomposition/_pls.py

```diff
@@ -587,6 +590,9 @@ class PLSRegression(_PLS):
     --------
     PLSCanonical : Partial Least Squares transformer and regressor.
 
+    For a comparison between PLS Regression and :class:`~sklearn.decomposition.PCA`, see
+    :ref:`sphx_glr_auto_examples_cross_decomposition_plot_pcr_vs_pls.py`.
```

Comment:
I think moving this to the `Examples` section directly bellow should fix the CI issue.